{ 'fall_position_front': { 'head': [ { 'duration': 1.0,
                                       'name': u'fall_position_front',
                                       'positions': [0.0, 0.0012],
                                       'starttime': 0.0}],
                           'left_arm': [ { 'duration': 1.0,
                                           'name': u'fall_position_front',
                                           'positions': [ -2.1621,
                                                          -1.4328,
                                                          0.0853,
                                                          0.0369,
                                                          -1.0434,
                                                          0.0672,
                                                          -0.0203],
                                           'starttime': 0.0}],
                           'left_leg': [ { 'duration': 1.0,
                                           'name': u'fall_position_front',
                                           'positions': [ 0.0063,
                                                          -0.0365,
                                                          -0.8302,
                                                          1.4101,
                                                          0.6402,
                                                          -0.0324],
                                           'starttime': 0.0}],
                           'right_arm': [ { 'duration': 1.0,
                                            'name': u'fall_position_front',
                                            'positions': [ 2.1621,
                                                           1.4328,
                                                           -0.0853,
                                                           -0.0369,
                                                           1.0434,
                                                           -0.0672,
                                                           0.0203],
                                            'starttime': 0.0}],
                           'right_leg': [ { 'duration': 1.0,
                                            'name': u'fall_position_front',
                                            'positions': [ -0.0063,
                                                           0.0365,
                                                           0.8302,
                                                           -1.4101,
                                                           -0.6402,
                                                           0.0324],
                                            'starttime': 0.0}],
                           'torso': [ { 'duration': 1.0,
                                        'name': u'fall_position_front',
                                        'positions': [-0.0003, -0.0067],
                                        'starttime': 0.0}]}}
